src/DefifaTiered721Delegate.sol

```diff
@@ -0,0 +1,138 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.6;
+
+import "@jbx-protocol/juice-nft-rewards/contracts/JBTiered721Delegate.sol";
+
+import "./structs/DefifaTierRedemptionWeight.sol";
+
+contract DefifaTiered721Delegate is JBTiered721Delegate {
+    //*********************************************************************//
+    // --------------------------- custom errors ------------------------- //
+    //*********************************************************************//
+
+    error INVALID_REDEMPTION_WEIGHTS();
+
+    //*********************************************************************//
+    // --------------------------- properties ---------------------------- //
+    //*********************************************************************//
+
+    // The total weight that can be divided among tiers
+    uint256 constant TOTAL_REDEMPTION_WEIGHT = 1_000_000_000;
+
+    // Tiers are limited to ID 100
+    uint256[100] tierRedemptionWeights;
```

Comment:
we should make the size 32 right?

Reply:
Im doing a lil experimenting here, since we want the gas cost to be low when getting the weight for a tier (like a mapping) but we also want the ability to easily reset all the values. Since you can't delete a mapping and you have to delete each key. I'll use the tests to see if this makes sense or not.

Reply:
Got it